### ISSUE TITLE
Add experimental design flow to VADS

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#85474]
## User Story
As a Governance team member supporting Collab Cycle, I want to add the revised experimental design flow to the VA Design System website so that VFS teams and other folks supporting VA.gov can understand the process.

Assignee: @allison0034 
Peer Reviewer: @it-harrison 

## Description
In ticket #81909, and then again in ticket #91693, Governance team put together a user flow (aka swim lane diagram and decision tree) on the experimental design process. The purpose of this was to ensure Governance guidance at Design Intent and Midpoint Review was in alignment with what is recommended to VFS teams re: the experimental design process. While the Governance team guidance on experimental design is now up to date and Governance is able to give VFS teams proper and accurate guidance on when to go through the experimental design process, we still believe it would be valuable for VFS teams to be able to visually take in this information themselves. The purpose of this ticket is to add both the swim lane diagram and the decision tree to the VA Design System website.

Please note that we will NOT be revising any of the guidance on VADS within the scope of this ticket. If content updates are necessary, please create a ticket, add the `governance-team` label, and link to it from the comments section of this ticket.

## Impacted Artifacts

[VADS - contributing to the Design System](https://design.va.gov/about/)

## Tasks
- [x] Determine where on VADS it would be valuable for the flow to be shared
- [x] The flow is much larger than the flow recently added to the Collab Cycle Overview page. Sharing a view-only version of the Mural may be best, however it would be nice if the link was a preview of the image. Is that possible? I think people are much more likely to check out the flow if they can _see_ that it's a flow. I do not think a link to the flow will suffice.
- [x] Alt text will likely be needed, so please check with Brian.
- [x] If content updates are needed to align the content to the newly defined flow, create a new ticket, tag `governance-team` and add to the comments section of this ticket
- [x] Once the flow is ready to be added, submit a PR and add @it-harrison and @caw310 as approvers
- [x] Verify the flow has been published to VADS

## Peer Review
_To be completed by peer reviewer_
- [x] User story and acceptance criteria are met

## Acceptance Criteria
- [x] We have added the new experimental design flow to the VADS website
- [x] We have created a ticket, if necessary, to update the experimental design content on VADS Website

## Team Notification
- [ ] Slack post on #platform-governance-team
- [ ] Presentation at Weekly Initiative Meeting
- [x] No notification needed

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [ ] Ticket has been moved to the Planned pipeline in the Governance team board
- [ ] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@allison0034 this is the ticket I mentioned yesterday. Do you think it makes sense to modify this ticket to include updating the content too? Or should we address that in another ticket?

**Comment by humancompanion-usds** [comment]
Here are the [updated flow diagrams](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1715279885465/a8a7701ed5948be8cd98f972ee930a16b9e7444b?sender=u9ae4d29a0ed5c3d2753a7824)


@allison0034 - It occurred to me yesterday when we were talking to the team working on VLC VoIP calls that there seems to be a disconnect in our diagram and understanding. There is work that is sometimes very clearly not going to be immediately applicable to other teams. In this case, while it is possible that another team might want to build in VoIP calls into their application, it's fairly unlikely (and even this one is limited by the Call Center being able to receive those calls). Thus there does need to be some judgement early on regarding the reusability of the design. Otherwise I fear the Experimental process will become a collection of 1-off designs that were never really meant to stand-alone outside of their application. Therefore the process could become essentially an index of all designs that contain a componet or pattern not currently in the Design System. That's too broad and unwieldy. We'd end up with hundreds of issues. Do you agree? If so, can we discuss revising the diagram one more time before updating the guidance?

**Comment by allison0034** [comment]
Hi @humancompanion-usds, that does make sense, however, experimental design was created so that when teams wanted to use a thing that was NOT in the design system we had a way to control and keep track of that. If we move to only rec experimental design when the thing is going to be reused, what do we do if teams are creating new components that are not going to be reused? Are they allowed to create new things or variations with no governance or approval?

The way the flow is set up currently for experimental design is in 2 steps.

1. The new thing needs to be researched and approved for use by DSC
2. If DSC feels it will be reused then it is added to the VADS

**Comment by humancompanion-usds** [comment]
> what do we do if teams are creating new components...?

If only! I've had exactly 2 teams submit new component code to the Design System in 2 years. If no other team is going to use a new component, why would it come into the Design System? There are going to be things that only make sense in a single application. However, they should be few and far between. Generally, things can, and should, be generalized enough to be reusable. 

> Are they allowed to create new things or variations with no governance or approval?

They would get governance from the Collab Cycle tracks. Not every unique piece of code has to be turned into a component and put into the Design System. Our system would just explode in size if that were the case.

**Comment by allison0034** [comment]
> They would get governance from the Collab Cycle tracks. Not every unique piece of code has to be turned into a component and put into the Design System. 

Governance team does not currently give any guidance around new or variations of components or patterns with the exception of saying you need to go thru experimental design. Brian may raise some accessibility concerns, but that is about it. Our role historically has been to say use what is in the VADS and if you are straying from that, or creating something new, go thru experimental design. @shiragoodman do you have anything to add?

**Comment by shiragoodman** [comment]
@allison0034 @humancompanion-usds I think this might be best to continue this conversation in person. Matt, FYSA Allison and I have been talking with @naomimarcussen and we have several concerns wrt changing the flow. I'd be happy to set up a time for us all to discuss. For now, I'll add to our 1:1 agenda for Friday 8/7 and we can discuss next steps.

**Comment by allison0034** [comment]
[Created PR](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/pull/3439)

**Comment by allison0034** [comment]
@governance-team new content ticket created #96032

**Comment by it-harrison** [comment]
@allison0034 - looks good!

**Comment by shiragoodman** [comment]
[This page](https://design.va.gov/about/contributing-to-the-design-system/) looks great @allison0034 ! Thank you. I really like the preview flow and it's very easy to see where to go to see the full sized version.
